tests/test_cellomics.py

```diff
@@ -0,0 +1,131 @@
+import os
+import numpy as np
+from skimage import io
+from collections import OrderedDict
+from microscopium.screens import cellomics
+import pytest
+import tempfile
+
+
+@pytest.fixture
+def image_files_25(request):
```

Comment:
Yes, you can probably make this happen:

```python

def image_files(num_files):
    @pytest.fixture
    def image_files_n(request):
        files = []
        for i in range(num_files):
        # [etc]
    return image_files_n
```

I *think* that should work. And it'd be beautiful! =)